certs.go

```diff
@@ -0,0 +1,46 @@
+package certs
```

Comment:
I moved the general "certs" types into a separate file away from the "k8s" types.